### ISSUE TITLE
Problem Clone()

### ISSUE THREAD

**Issue by muukii** [delba/TextAttributes#5]
``` swift
let attribute1 = TextAttributes().font(UIFont(name: "AvenirNext", size: 12))

let attribute2 = attribute1.clone()

attribute2.font  // => UIFont(name: "HelveticaNeue", size: 12) 😥
attribute2.dictionary[NSFontAttributeName] // => UIFont(name: "AvenirNext", size: 12)
```

I think that there is better to change StoredProperty to ComputedProperty.
How do you think?

**Comment by delba** [comment]
Oh indeed 😱 

I don't like the idea of changing the stored properties to computed ones though.
Indeed, we would have to cast the value every time we want to access a property like `attrs.font`.

I would prefer to do the ugly stuff in a specific place (like a function) that we will call only when cloning.

**Comment by muukii** [comment]
@delba 
I understand your concern. 
Are you looking to fix this in some other way then?

**Comment by delba** [comment]
I'll probably fix that during the weekend :smiley:

**Comment by muukii** [comment]
Thank you so much!!!

**Comment by delba** [comment]
Hi @muukii,

It has been fixed in `0.2.4`.

Thanks for the issue!

**Comment by muukii** [comment]
@delba thanks!
